### ISSUE TITLE
Could #220 be released please?

### ISSUE THREAD

**Issue by benwainwright** [typescript-language-server/typescript-language-server#239]
https://github.com/theia-ide/typescript-language-server/pull/220 has been merged into master but not yet actually released. 10 days ago the Yarn documentation was amended to say that typescript-language-server should work out of the box, which I suspect it won't do until the above change is released. This was obviously a hasty move by contributors in the yarn space and is pretty confusing for users.

Are you expecting to release a new version soon? I could really do with this change...

**Comment by rchl** [comment]
We need help from @DonnieWest who is the one that has permissions for releasing new version.

**Comment by benwainwright** [comment]
Only one single person can do releases? 🤪

**Comment by benwainwright** [comment]
I've now emailed @DonnieWest direct from the email address published on his website

**Comment by rchl** [comment]
I did that a couple of days too. It's a little hard to get ahold of Donnie but if does seem to work eventually. :)

Hopefully we'll address that situation eventually.
The backup plan is to ask the original authors.

**Comment by DonnieWest** [comment]
Howdy y'all

Apologies for the silence, I've been on vacation. This situation has revealed three problems that I intend to fix:

1. I'm the only one able to perform a release

In addition to giving @rchl NPM permissions, I'm going to make releases happen through the existing CI system. This way, anyone who can publish a Github release can also publish an NPM release. This should also clean up our error prone release process

2. I don't have an open channel of communication with everyone

We probably need a (functional) IRC channel or something like Matrix/Slack/Discord so we can ask questions and I have something a bit more real time to chat with people through. I'm open to anything, but I'd prefer something open source.

3. This still appears to be part of typefox and theia-ide and it's not

They're no longer affiliiated here, so I'll edit the README to provide proper attribution. Additionally, I'll move us to a new Github organization to ensure they're not pinged for my negligence :P


As long as there are no objections, I'll get rolling on cutting a release and making these changes :)

**Comment by rchl** [comment]
> I'm going to make releases happen through the existing CI system. This way, anyone who can publish a Github release can also publish an NPM release. This should also clean up our error prone release process

I hope you can make it work from CI. Though that would probably require the 2FA to be disabled for the user that is gonna be used.

> We probably need a (functional) IRC channel or something like Matrix/Slack/Discord so we can ask questions and check on releases. I'm open to anything, but I'd prefer something open source.

I'm already on Discord as many projects use that. I'm not really aware of any open-source alternative and there are any, those are probably pretty niche.

**Comment by polyzen** [comment]
Matrix is the open-source alt. If you were to create a channel on Libera.Chat (IRC) you would be able to connect to it from a Matrix bridge hosted by https://matrix.org:
https://matrix-org.github.io/matrix-appservice-irc/latest/bridged_networks.html

**Comment by DonnieWest** [comment]
#220 has been released. Additionally, I think things are set up so people can communicate effectively and aren't blocked on me anymore. Congrats everyone :)
